src/components/pages/collectRecordFormPages/CollectRecordFormPageAlternative/CollectRecordFormPageAlternative.js

```diff
@@ -84,7 +85,7 @@ function loadObservationsFromCollectRecordIntoTableState({
     setObservationsTableReducerInitialized(true)
   }
 
-  if (!collectRecordBeingEdited) {
+  if (isNewRecord) {
```

Comment:
This was me being sloppy. `!collectRecordBeingEdited` will happen sometimes when the component first initializes (`collectREcordBeingEdited` will be undefined initially`)